build.gradle

```diff
@@ -49,29 +107,61 @@ task checkstyle(type: Checkstyle) {
     source = sourceSets.main.allJava + sourceSets.test.allJava
 }
 
-task gem(type: JRubyExec, dependsOn: ["build", "gemspec", "classpath"]) {
-    jrubyArgs "-rrubygems/gem_runner", "-eGem::GemRunner.new.run(ARGV)", "build"
-    script "build/gemspec"
-    doLast { ant.move(file: "${project.name}-${project.version}.gem", todir: "pkg") }
+java {
+    withJavadocJar()
+    withSourcesJar()
+}
+
+jar {
+    from rootProject.file("LICENSE")
+}
+
+sourcesJar {
+    from rootProject.file("LICENSE")
+}
+
+javadocJar {
+    from rootProject.file("LICENSE")
 }
 
-task gemspec << { file("build/gemspec").write($/
-Gem::Specification.new do |spec|
-  spec.name          = "${project.name}"
-  spec.version       = "${project.version}"
-  spec.authors       = ["Manabu Takayama"]
-  spec.summary       = %[S3 file output plugin for Embulk]
-  spec.description   = %[Stores files on S3.]
-  spec.email         = ["learn.libra@gmail.com"]
-  spec.licenses      = ["MIT"]
-  spec.homepage      = "https://github.com/llibra/embulk-output-s3"
+//signing {
+//    // https://docs.gradle.org/current/userguide/signing_plugin.html#sec:using_gpg_agent
+//    useGpgCmd()
+//    sign publishing.publications.maven
+//}
 
-  spec.files         = `git ls-files`.split("\n") + Dir["classpath/*.jar"]
-  spec.test_files    = spec.files.grep(%r"^(test|spec)/")
-  spec.require_paths = ["lib"]
+publishing {
+    publications {
+        embulkPluginMaven(MavenPublication) {  // Publish it with "publishEmbulkPluginMavenPublicationToMavenRepository".
+            from components.java  // Must be "components.java". The dependency modification works only for it.
+        }
+    }
+    repositories {
+        maven {
+            url = "${project.buildDir}/mavenPublishLocal"
+        }
+    }
+}
+
+embulkPlugin {
+    mainClass = "org.embulk.output.S3FileOutputPlugin"
+    category = "output"
+    type = "s3"
+}
 
-  spec.add_development_dependency 'bundler', ['~> 1.0']
-  spec.add_development_dependency 'rake', ['>= 10.0']
-end
-/$)
+gem {
+    from("LICENSE.txt")  // Optional -- if you need other files in the gem.
```

Comment:
Let's remove this for the same reason with the above.

Reply:
Thanks. Removed